package.json

```diff
@@ -21,29 +21,28 @@
   },
   "homepage": "https://github.com/twbs/stylelint-config-twbs-bootstrap#readme",
   "files": [
-    "css/index.js",
-    "scss/index.js"
+    "index.js"
   ],
   "engines": {
-    "node": ">=10"
+    "node": ">=12"
   },
   "peerDependencies": {
-    "stylelint": "^13.13.1"
+    "stylelint": "^14.1.0"
```

Comment:
This doesn't need to be 14.1.0; 14.0.0 is fine.

Reply:
Per @jeddy3's comment in the issue thread, we do need 14.1 otherwise the overrides syntax won't work 100% with the `extends` syntax.

Reply:
Yup, I thought about it after commenting. It's all good 👍